### PR TITLE
Some Minor Grammatical & Styling Changes

### PR DESCRIPTION
**Pull request by TheoBearman**

### CODE REVIEW

File: resources/lang/en/forms.php [hunk 1]
```diff
@@ -94,17 +94,17 @@
             'site-name'              => 'Site Name',
             'site-url'               => 'Site URL',
             'display-graphs'         => 'Display graphs on status page?',
-            'about-this-page'        => 'About this page',
-            'days-of-incidents'      => 'How many days of incidents to show?',
+            'about-this-page'        => 'About This Page',
+            'days-of-incidents'      => 'How Many Days of Incidents to Show?',
```
Comment (jbrooksuk) [inline resources/lang/en/forms.php]: I don't like these kind of changes.

**Comment by GrahamCampbell** [comment]
This needs to be consistent across all the languages.

**Comment by jbrooksuk** [comment]
> This needs to be consistent across all the languages.

:trollface: easier said than done ;)

File: resources/lang/en/forms.php [hunk 2]
```diff
@@ -42,12 +42,12 @@
         'component'          => 'Component',
         'message'            => 'Message',
         'message-help'       => 'You may also use Markdown.',
-        'scheduled_at'       => 'When to schedule the maintenance for?',
-        'incident_time'      => 'When did this incident occur?',
-        'notify_subscribers' => 'Notify subscribers',
+        'scheduled_at'       => 'When to Schedule the Maintenance For?',
```
Comment (jbrooksuk) [inline resources/lang/en/forms.php]: This shouldn't be changed.

File: resources/lang/en/forms.php [hunk 3]
```diff
@@ -42,12 +42,12 @@
...
+        'incident_time'      => 'When Did This Incident Occur?',
```
Comment (jbrooksuk) [inline resources/lang/en/forms.php]: THis shouldn't be changed.

File: resources/lang/en/forms.php [hunk 4]
```diff
@@ -42,12 +42,12 @@
...
+        'notify_subscribers' => 'Notify Subscribers?',
         'visibility'         => 'Incident Visibility',
-        'public'             => 'Viewable by public',
-        'logged_in_only'     => 'Only visible to logged in users',
+        'public'             => 'Viewable By Public',
+        'logged_in_only'     => 'Only Visible to Logged in Users',
```
Comment (jbrooksuk) [inline resources/lang/en/forms.php]: This shouldn't be changed.

File: resources/lang/en/forms.php [hunk 5]
```diff
@@ -76,8 +76,8 @@
         'description'      => 'Description',
         'description-help' => 'You may also use Markdown.',
         'display-chart'    => 'Display chart on status page?',
-        'default-value'    => 'Default value',
-        'calc_type'        => 'Calculation of metrics',
+        'default-value'    => 'Default Value',
+        'calc_type'        => 'Calculation of Metrics',
```
Comment (jbrooksuk) [inline resources/lang/en/forms.php]: This shouldn't be changed.

File: resources/lang/en/forms.php [hunk 6]
```diff
@@ -101,10 +101,10 @@
             'subscribers'            => 'Allow people to signup to email notifications?',
         ],
         'analytics' => [
-            'analytics_google'       => 'Google Analytics code',
-            'analytics_gosquared'    => 'GoSquared Analytics code',
-            'analytics_piwik_url'    => 'URL of your Piwik instance (without http(s)://)',
-            'analytics_piwik_siteid' => 'Piwik\'s site id',
+            'analytics_google'       => 'Google Analytics Code',
+            'analytics_gosquared'    => 'GoSquared Analytics Code',
+            'analytics_piwik_url'    => 'URL of your Piwik Instance (without http(s)://)',
```
Comment (jbrooksuk) [inline resources/lang/en/forms.php]: This shouldn't be changed.

File: resources/lang/en/forms.php (hunk 2 above)
Comment (TheoBearman) [inline resources/lang/en/forms.php]: @jbrooksuk What specifically?

File: resources/lang/en/forms.php (hunk 3 above)
Comment (TheoBearman) [inline resources/lang/en/forms.php]: @jbrooksuk What specifically?

**Comment by AntoineAugusti** [comment]
It doesn't make sense to put uppercase letters at the beginning of every words in French, so, not sure it should be consistent for all the languages

**Comment by GrahamCampbell** [comment]
It doesn't really make sense in English tbh.

**Comment by GrahamCampbell** [comment]
It only makes sense for titles.

**Comment by jbrooksuk** [comment]
Perfect! :)

**Pull request merged**